quetz/main.py

```diff
@@ -23,10 +23,12 @@
 from zipfile import ZipFile
 import zstandard
 
-from quetz import auth_github
 from quetz import config
+config.load_configs()
```

Comment:
You're right, it's not clean.
At this time, some `Quetz` modules use variables from config (in [auth_github.py](https://github.com/TheSnakePit/quetz/blob/d9cd6dbd973939b246226489ad8143ff9b505b68/quetz/auth_github.py#L20) for example) because they were declared at import time in globals.

I will propose improvements